File: VB/WebSite/clsName.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text


	Public Class clsName
		Private _Code As String = ""
		Private _FirstName As String = ""
		Private _LastName As String = ""

		Public Sub New(ByVal code As String, ByVal firstName As String, ByVal lastName As String)
			_Code = code
			_FirstName = firstName
			_LastName = lastName
		End Sub

		Public ReadOnly Property Code() As String
			Get
				Return _Code
			End Get
		End Property

		Public ReadOnly Property FirstName() As String
			Get
				Return _FirstName
			End Get
		End Property

		Public ReadOnly Property LastName() As String
			Get
				Return _LastName
			End Get
		End Property
	End Class

File: CS/WebSite/clsName.cs
using System;
using System.Collections.Generic;
using System.Text;


    public class clsName
    {
        private string _Code = "";
        private string _FirstName = "";
        private string _LastName = "";

        public clsName(string code, string firstName, string lastName)
        {
            _Code = code;
            _FirstName = firstName;
            _LastName = lastName;
        }

        public string Code
        {
            get { return _Code; }
        }

        public string FirstName
        {
            get { return _FirstName; }
        }

        public string LastName
        {
            get { return _LastName; }
        }
}
